info2439/module6/assignment5.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Camden Grove | INFO 2439 | map/location</title>
+  <meta charset="utf-8"> 
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css"/>
+	<link href="https://fonts.googleapis.com/css?family=Notable&display=swap" rel="stylesheet">
+		<link rel="stylesheet" href="Mod5_css/cgrove.css">
+<!-- theme roller  -->
+<link rel="stylesheet" href="theme/themes/cth.css" />
+  <link rel="stylesheet" href="theme/themes/jquery.mobile.icons.min.css" />
+<!-- theme roller-->
+  <script src="https://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="https://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+    
+</head>
+<body>
+<!-- Camden Grove
+     Assignment3.html
+     INFO2439
+     Thoendel
+     10/12/19
+-->
+<div data-role="page"><!-- page div-->
+    <div data-role="header" data-position="fixed"><!--header div -->
+	<a data-ajax="false" href="http://student.mccinfo.net/~cjgrove/info2439/index.html" class="ui-btn-left" target="_blank" data-icon="home">Home</a><h1>Camden Grove</h1></div>
+
+<h1>Assignment #5<br>Sage Bistro Static Google Map</h1>
+	  <div role="main" class="ui-content"><!--content div -->
+<div class="ui-grid-a">
+<div class="ui-block-a">
+<img alt="map" src="https://maps.googleapis.com/maps/api/staticmap?center=Sage+Student+Bistro,Omaha,NE&zoom=17&size=150x150&maptype=roadmap&markers=color:blue%7Clabel:S%7C40.41.304729,-95.959506&key=AIzaSyAFQid4GkVK5cIqrk3n8kuRkAo7YcNeTzo">
```

Comment:
This looks like you used my key. So - I would expect to see screenshots in your submission where you showed the creation of your own keys without enabling billing.